release-notes/swan-lake-beta2-release-note.md

```diff
@@ -40,6 +40,16 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta2%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
 
+Set the value type of the string iteration sequence to string:Char.
```

Comment:
Let's move this to the "Improvements" section.

It is only the type that changed, right? Shall we say

```suggestion
- The static type of string iteration has been changed from `string` to `string:Char`.
```

Reply:
Yes, I'll change it.